packages/celotool/src/lib/helm_deploy.ts

```diff
@@ -188,9 +203,11 @@ export async function installCertManagerAndNginx(
   const nginxChartVersion = '3.9.0'
   const nginxChartNamespace = 'default'
 
-  // Cert Manager is the newer version of lego
+  // Check if cert-manager is installed in any namespace
+  // because cert-manager crds are global and cannot live
+  // different crds version in the same cluster
   const certManagerExists = await outputIncludes(
-    `helm list -n default`,
+    `helm list -A`,
```

Comment:
I can update this. I have updated to `cert-manager` namespace because this note in their documentation
![image](https://user-images.githubusercontent.com/5635989/110817921-12107480-828d-11eb-8c8b-8921cedf7feb.png).

And because updating the namespace, it could happen some issues for cluster that already have it deployed. But I did not think about the RBAC restricted deployment. I will update to handle that.